superset/db_engine_specs/couchbase.py

```diff
@@ -74,14 +74,15 @@ class CouchbaseParametersSchema(Schema):
     )
 
 
-class CouchbaseDbEngineSpec(BasicParametersMixin, BaseEngineSpec):
-    engine = "couchbasedb"
+class CouchbaseEngineSpec(BasicParametersMixin, BaseEngineSpec):
+    engine = "couchbase"
+    engine_aliases = {"couchbase", "couchbasedb"}
```

Comment:
You donät need the main name in the alias set, this should be enough:
```suggestion
    engine_aliases = {"couchbasedb"}
```

Reply:
Updated

Reply:
There are 2 failed checks, but they were not failing earlier, are they because of my patch ?

Reply:
The failed MySQL test is flaky - we can restart the test. The docs one seems unrelated - It seems `resources/INTHEWILD.md` has a link to `/www.cnovit.com` that isn't working. I think we may need to remove the link if the homepage isn't up anymore. Do you want to do it in this PR? I can also open a PR to fix it if needed.

Reply:
Fixed